linera-service/src/node_service.rs

```diff
@@ -39,32 +42,73 @@ use linera_storage::Store;
 use linera_views::views::ViewError;
 use serde::{Deserialize, Serialize};
 use serde_json::json;
-use std::{net::SocketAddr, num::NonZeroU16, sync::Arc};
+use std::{collections::BTreeMap, net::SocketAddr, num::NonZeroU16, sync::Arc};
 use thiserror::Error as ThisError;
```

Comment:
This is weird: we're using `thiserror:ThisError` in place of `anyhow::Error` ?

Reply:
That's just to derive the `Error` trait (and `Display` and stuff) for `NodeServiceError`. So I'd say it's better than using `anyhow::Error`?